packages/backup-restore/src/firestore/restoreService.ts

```diff
@@ -0,0 +1,140 @@
+import admin, { firestore } from "firebase-admin";
+
+import { Collection } from "@eisbuk/shared";
+import {
+  IOperationFailure,
+  IOperationSuccess,
+  IOrgRootData,
+  IOrgData,
+  ISubCollectionData,
+} from "../lib/types";
+
+//* * #region restoreScripts */
+
+/**
+ * restoreOrganizations - Set all data for an array of organizations
+ * @param {IOrgData[]} orgs - An array of organizations to write: { id, data, subCollections }
+ */
+export async function restoreOrganizations(
+  orgs: IOrgData[]
+): Promise<IOperationSuccess<string> | IOperationFailure> {
+  try {
+    const writeOrgOps = orgs.map(async (org) => {
+      await setOrganization(org);
+    });
+
+    await Promise.all(writeOrgOps);
+
+    return { ok: true, data: "OK" };
+  } catch (err: any) {
+    return { ok: false, message: err.message };
+  }
+}
+
+//* #endregion restoreScripts */
+
+//* * #region restoreUtils */
```

Comment:
Write region comments in any way you like, but when using `// #region <regionName>` syntax, VSCode lets you minimise the region (by clicking an arrow on the left hand side, next to line numbers)